README.md

```diff
@@ -51,13 +52,23 @@ but was skipped because it was published from the wrong commit of
 `draft-ietf-ppm-dap-07` is effectively identical to draft 6, but with those bugs
 fixed.
 
+`draft-ietf-ppm-dap-08` was also skipped, since it contained only minor
+mechanical protocol changes and was not implemented or deployed by anyone else.
+
 [dap-01]: https://datatracker.ietf.org/doc/draft-ietf-ppm-dap/01/
 [dap-02]: https://datatracker.ietf.org/doc/draft-ietf-ppm-dap/02/
 [dap-03]: https://datatracker.ietf.org/doc/draft-ietf-ppm-dap/03/
 [dap-04]: https://datatracker.ietf.org/doc/draft-ietf-ppm-dap/04/
 [dap-07]: https://datatracker.ietf.org/doc/draft-ietf-ppm-dap/07/
+[dap-09]: https://datatracker.ietf.org/doc/draft-ietf-ppm-dap/09/
 [dap-gh]: https://github.com/ietf-wg-ppm/draft-ietf-ppm-dap
 
+### `draft-ietf-ppm-dap-07` errata
+
+There is a bug in Janus' `release/0.6` branch wherein `PrepareResp` messages are
+incorrectly encoded, making Janus incompatible with other DAP implementations.
```

Comment:
It might be worth pointing out that this is only a problem for aggregator-aggregator communication.
```suggestion
incorrectly encoded, making Janus incompatible with other DAP Aggregator
implementations.
```